apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  sentHeader: {
+    css: '.alert-message'
+  },
+  content: {
+    email: 'test@test.com'
```

Comment:
if you rename this to 'email-address' you can use it directly in the feature file:
```js
yield I.setSessionData(steps.name, {
  confirmationPage.content
});
```